modules/contact/doc/content/documentation/systems/Problem/ReferenceResidualProblem.md

```diff
@@ -50,6 +50,14 @@ and the temperature `saved` variable is applied in the heat conduction kernel
 
 !listing modules/combined/test/tests/reference_residual/reference_residual.i start=Kernels end=Functions
 
```

Comment:
Would you add a third level heading here, like `### Generalized Plane Strain Use` or something similar to separate out this new section a bit more clearly please?